core/templates/dev/head/services/SearchService.ts

```diff
@@ -16,206 +16,230 @@
  * @fileoverview search service for activityTilesInfinityGrid
  */
 
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { HttpClient } from '@angular/common/http';
+import { EventService } from './EventService';
```

Comment:
Add another newline above this, we're going to split all third party imports and Oppia's.